packages/fx-core/templates/plugins/resource/simpleauth/bicep/simpleAuth.template.bicep

```diff
@@ -0,0 +1,64 @@
+param sku string
+param simpleAuthServerFarmsName string
+param simpleAuthWebAppName string
+param tenantId string
```

Comment:
update the name or add a description here to mention it's the M365 tenant id, not Azure tenant id?

Reply:
Fixed. Renamed to `m365TenantId`